lib/ansible/modules/cloud/amazon/cloudformation_stack_set.py

```diff
@@ -429,14 +428,15 @@ def await_stack_instance_completion(module, cfn, stack_set_name, max_wait):
 
 
 def await_stack_set_exists(cfn, stack_set_name):
-    # AWSRetry will retry on `NotFound` errors for us
+    # AWSRetry will retry on `StackSetNotFound` errors for us
```

Comment:
Is there another PR to add StackSetNotFound to AWSRetry or should this have a `catch_extra_error_codes`?

Reply:
line 539?

Reply:
yep, it's handled by the module

Reply:
I think this should be good to go... but I'm still running the full test suite with ci_complete. I've run it about 5 times, but would like to double that to have a better chance of finding flakiness I missed. (So far, haven't seen any failures due to something I've missed)

Reply:
Ok thank you, my eyes were just not seeing it.